Resources/config/form-type.xml

```diff
@@ -9,5 +9,13 @@
             <tag name="form.type" alias="cmf_core_checkbox_url_label" />
             <argument type="service" id="router" />
         </service>
+
+        <service id="cmf_core.form.extension.publishable" abstract="true" class="Symfony\Cmf\Bundle\CoreBundle\Form\Extension\PublishableExtension">
+            <!-- <argument /> Extended type fqcn -->
```

Comment:
wouter, fbourigault had the <argument/> outside the comment and i proposed to put it inside to make usage of the service template easier.

can the extension accept an array of extended types? then yes, we should use a configuration.

if that is not possible, i think this is fine as is. alternatively, we could still have configuration for the types and copy the service in the DI extension for each type to extend.